pep-0008.txt

```diff
@@ -121,6 +121,51 @@ Optional::
       var_one, var_two,
       var_three, var_four)
 
+For dictionaries there is an extra rule. When placing a dictionary key on a new
+line an additional hanging indent should be added. This is so it is easy to
+differentiate between keys and values.
+
+Yes::
+
+    # Keys aligned with opening delimiter, values have an extra indent.
+    mydict = {'firstkey':
+                  'a very very very very very long value',
+              'secondkey': 'a short value',
+              'thirdkey': 'a very very very'
+                  ' long value that continues on the next line'
+                  ' and even on the line after that.',
+    }
+
+    # Keys with one and values with two levels of hanging indents.
+    mydict = {
+        'firstkey':
+            'a very very very very very long value',
+        'secondkey': 'a short value',
+        'thirdkey': 'a very very very'
```

Comment:
Just to share what other IDE is already doing with respect to this issue.

PyCharm will automatically do this for you
```
'thirdkey':
    'a very very very'
    'long value that continues on the next line'
    'and even on the line after that.',
```
And this too
```
'thirdkey': 'a very very very'
            'long value that continues on the next line'
            'and even on the line after that.',
```
But it will not do this one, which is being considered as a 'yes' in this proposal.
```
'thirdkey': 'a very very very'
    'long value that continues on the next line'
    'and even on the line after that.',
```
Perhaps something to take into account when making decision about this.
Thanks :)